doc/Type/Any.pod6

```diff
@@ -1051,13 +1051,26 @@ Defined as:
     multi method reduce(Any:D: &with)
     multi sub reduce (&with, +list)
 
-Applying it to a class will always produce Nil.
-Applies its argument (or first argument, in case it's a sub) as an operator to
-all the elements in the object (or second argument), producing a single result.
-The argument must be an infix operator or take, in any case, two positional
-arguments.
+This routine combines the elements in a list-y object together, and thus
+produces a single result, by using a binary subroutine. The subroutine must be
+either an
+L<infix operator|https://docs.raku.org/language/operators#index-entry-infix_operator>
+or take two positional arguments.
 
-    (1..13).reduce( &[*] ).say; # OUTPUT: «6227020800␤»
+    say (1..4).reduce( &[+] );     # OUTPUT: «10␤»
+    say reduce &[+], 1..4;         # OUTPUT: «10␤»
+
```

Comment:
This is not exactly what you describe, and I think it might be some subproduct of the hyper operator, which is what you're applying here. If you're going to use `+`, use the `infix<+>` definition, not the hyper. Also mention somewhere that it's probably better to use that latter.

Reply:
>If you're going to use +, use the infix<+> definition, not the hyper. Also mention somewhere that it's probably better to use that latter.

I left it out from this page since I think it's better discussed in the `List` page where it's introduced. However, I didn't know how to word it so I didn't mention there either. Probably better for another PR.